include/core/parameters.h

```diff
@@ -248,6 +220,38 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief Fluid - Class for fluid definition
+   */
+  class Fluid
+  {
+  public:
+    Fluid()
+    {}
+
+    void
+    declare_parameters(ParameterHandler &prm, unsigned int id);
+    void
+    parse_parameters(ParameterHandler &prm, unsigned int id);
+
+    // Kinematic viscosity (nu = mu/rho) in units of L^2/s
+    double viscosity;
+    // volumetric mass density (rho) in units of kg/m^3
+    double density;
+    // specific heat capacity (cp) in J/K/kg
+    double specific_heat;
+    // thermal conductivity (k) in W/m/K
```

Comment:
```suggestion
    // thermal conductivity (k) in W/m.K
```

Reply:
Not a good idea, use parenthesis if you do that.